core/services/keystore/keys/ocr2key/key_bundle.go

```diff
@@ -72,6 +75,8 @@ func MustNewInsecure(reader io.Reader, chainType chaintype.ChainType) KeyBundle
 		return mustNewKeyBundleInsecure(chaintype.Solana, newSolanaKeyring, reader)
 	case chaintype.StarkNet:
 		return mustNewKeyBundleInsecure(chaintype.StarkNet, starkkey.NewOCR2Key, reader)
+	case chaintype.Aptos:
```

Comment:
Is this `new<>Keyring` method still necessary? Starknet seems to have moved to use `NewOCR2Key` directly

\cc @jmank88 (you've previously moved the starknet code into core)

Reply:
![no-memory](https://github.com/smartcontractkit/chainlink/assets/1194128/44182775-a619-4344-9887-31a227f91226)
I don't know off the top of my head but it seems worth exploring.